src/tactic/hint.lean

```diff
@@ -50,6 +50,7 @@ add_hint_tactic "dsimp at *"
 add_hint_tactic "simp at *" -- TODO hook up to squeeze_simp?
 attribute [hint_tactic] tidy.ext1_wrapper
 add_hint_tactic "fsplit"
+add_hint_tactic "constructor"
```

Comment:
```suggestion
add_hint_tactic "fconstructor"
```
In all cases where it matters (i.e. when it generates a goal depending on another it generates), I think `fconstructor` is a lot friendlier than `constructor`.

Reply:
Then again, if we call `fconstructor`, I think we can remove `fsplit`, which is just a more restricted version.

Reply:
Let's remove `fsplit`, then.